### PR TITLE
LPS-191797 Change appearance of file selector

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by jkappler** [comment]
backend looks good @sandrodw3 :P

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx [hunk 1]
```diff
@@ -177,7 +177,7 @@ function Import({backURL, importURL, portletNamespace}: Props) {
 							{Liferay.Language.get('import-file')}
 						</h2>
 
-						<p>
+						<p className="c-mb-4 text-secondary">
```
Comment (p2kmgcl) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:184]: I think we can replace that with a `gap-4` on the ClayLayout.Sheet :smile:

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx [hunk 2]
```diff
@@ -32,6 +31,8 @@ function Import({backURL, importURL, portletNamespace}: Props) {
 		setImportResults,
 	] = useState<ImportResultsData | null>(null);
 
+	const fileInputRef = useRef() as React.MutableRefObject<HTMLInputElement>;
```
Comment (p2kmgcl) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:34]: useRef is a generic, we can (and should) write `useRef<HTMLInputElement>()` here

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx [hunk 3]
```diff
@@ -190,21 +191,30 @@ function Import({backURL, importURL, portletNamespace}: Props) {
...
-									className="reference-mark"
-									symbol="asterisk"
-								/>
+								{Liferay.Language.get('file-upload')}
 							</label>
 
-							<ClayInput
+							<input
+								accept={ZIP_EXTENSION}
 								data-testid={`${portletNamespace}file`}
```
Comment (p2kmgcl) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:199]: Not related to this PR, but I am not a fan of data-testid, specially if we already have labels/ids to access these elements.

**Review by p2kmgcl — CHANGES_REQUESTED** [review]

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx (hunk 3 above)
Comment (sandrodw3) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:199]: Me neither 😔 it was already there. Now that I think, I'll take advantage to remove in this PR.

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:184]: But the sheet doesn't have flex :disappointed:

Comment (sandrodw3) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:184]: Done, I had to add `c-mb-0` to all children but I think it's better anyway 😄

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx [hunk 4]
```diff
@@ -45,10 +47,10 @@ function Import({backURL, importURL, portletNamespace}: Props) {
 		setFileName(fileName);
 
 		const fileExtension = fileName
-			.substring(fileName.lastIndexOf('.') + 1)
+			.substring(fileName.lastIndexOf('.'))
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:51]: Yeah, it was done by @claraizquierdo 🥳

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:34]: Done 😄 thanks!

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx (hunk 3 above)
Comment (sandrodw3) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx:199]: Changed ✅

**Comment by sandrodw3** [comment]
@p2kmgcl thanks for the review!! all changes applied 🥳

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/Import.tsx]: :cry: it's true, we need the mb-0

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**